app/components/works/state_display_component.rb

```diff
@@ -9,6 +9,7 @@ class StateDisplayComponent < ApplicationComponent
         'first_draft' => 'Draft - Not deposited',
         'version_draft' => 'New version draft - Not deposited',
         'pending_approval' => 'Pending approval - Not deposited',
```

Comment:
@amyehodge question as to whether you want this full string on the right to be displaying or not -- @mjgiarlo is hoping to normalize how we display state info.

Reply:
I would say the "Not deposited" portion of the text does not need to display. And if I think about it, I wonder if that might make some people think that the "New version draft" somehow caused the original to be un-deposited? (not a thing, but they get very confused about the workflow sometimes, which is understandable).

Reply:
This will be changing per post-stand-up discussion. Stay tuned.

Reply:
@amyehodge for each of the six states here, could you write up the label you'd like to see in the app in a new issue, pls? It won't be a huge change, and it'd be good to get these right. Thanks!